apps/test/unit/templates/sectionsRefresh/QuickAssignTableTest.jsx

```diff
@@ -84,8 +76,16 @@ describe('QuickAssignTable', () => {
       updateCourse: () => {},
       sectionCourse: {displayName: 'Computer Science A', courseOfferingId: 73},
     };
-    const wrapper = mount(<QuickAssignTable {...props} />);
-    const radio = wrapper.find("input[value='Computer Science A']");
-    expect(radio.props().checked).toBe(true);
+    render(<QuickAssignTable {...props} />);
+    const radio = screen.getByLabelText('Computer Science A');
+    expect(radio).toBeChecked();
+  });
+
+  it('shows TA icon when course offering has TA enabled', () => {
```

Comment:
this is the only new test case in this file, the others are all just refactored to use RTL.